builders/manage/dapp/manage-tokens.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Manage Tokens
+description: Learn how the appchain governor can manage the token of a Tanssi-powered appchain using the dApp to mint them, configure the EIP-1559 fee market, and more.
+---
+
+# Manage Tokens
+
+## Introduction {: #introduction }
+
+Any appchain deployed through Tanssi is sovereign and free to define the governance model that best fits its use case. The appchain governor has superpowers over the chain administration, meaning privileged functions, such as updating the runtime, fall within his reach.
+
+There are some actions related to the native token management that are available to the appchain governor on the Tanssi dApp:
+
+- **Mint Tokens** - mints new tokens, increasing the total supply
+- **Update Balances** - increases or decreases the balance of an account, affecting the total supply
+- **Transfer Tokens** - executes a forced token transfer from one account to another
+- **Configure Gas Dynamics** - only available on EVM-compatible appchains, this action changes the [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md){target=\_blank} configuration, affecting the transaction pricing mechanism
+
+In this guide, you'll learn how to execute the previously listed actions using the [Tanssi dApp](https://apps.tanssi.network){target=\_blank}.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+- A Tanssi appchain (Snap or Dedicated)
+- The account you used when registering the appchain, imported in any of the [supported wallets](/builders/deploy/dapp/#supported-wallets){target=\_blank}
+- Your appchain's Sudo account, also imported in any of the [supported wallets](/builders/deploy/dapp/#supported-wallets){target=\_blank}. 
+
+!!! note
+    The appchain's registration account is always a Substrate one, whereas the appchain's Sudo account depends on the chain type. If the chain is EVM-compatible, then the Sudo account will be an Ethereum type and, otherwise, a Substrate type.
+
+### Retrieving the Registration Account {: #retrieving-registration-account }
+
+If you're unsure what your registration account is, you can query it directly from the Tanssi orchestrator chain, which keeps records of every registered appchain. To do so, head to the **Chain state** section on the Polkadot.js Apps connected to the orchestrator chain for [snap appchains](https://polkadot.js.org/apps/?rpc=wss://fraa-flashbox-2607-rpc.a.stagenet.tanssi.network#/chainstate){target=\_blank} or [dedicated appchains](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, and take the following steps:
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert your appchain id
+4. Press **+** icon
+
+You'll see the registration account at the bottom.
+
+![Locating your registration account](/images/builders/manage/dapp/token-management/token-management-1.webp)
```

Comment:
I'm fine with this being here for now due to the sake of time. But I wonder if we should extract this to it's own short guide and just link back to it as needed. What do you think?

Reply:
Agreed. The same applies to "retrieving the sudo account".
I'll create a ticket to make these sections a snippet at least